tests/functional/splash-example.js

```diff
@@ -0,0 +1,52 @@
+/*globals casper */
+
+'use strict';
+
+casper.test.begin('Test FormatJS home page splash example', function (test) {
+
+    test.comment('Load home page');
+    casper.start(casper.host + '/', function () {
+        test.pass('Page was loaded');
+    });
+
+    casper.then(function () {
+        test.comment('Test splash example');
+
+        test.assertExists('.splash-example-container', 'Found splash example container');
+        test.assertExists('.num-photos-select', 'Found number select');
+        test.assertExists('.locale-select', 'Found locale select');
+        test.assertExists('.splash-example-output', 'Found splash example output');
+
+        var output = casper.evaluate(function (numPhotos, locale) {
+            // Start by setting the takenDate prop to a known value so that we
+            // can compare the example's output to a static value.
+            var container = document.querySelector('.splash-example-container');
+            var component = container.component;
+            component.setProps({ takenDate: Date.now() });
```

Comment:
Well instead we could simply pass the static value directly in here instead of going through he mocked `Date.now()`.